src/Pension/BeamtenPensionRechner.php

```diff
@@ -2,14 +2,27 @@
 
 namespace Finanzrechner\Pension;
 
-use function Dgame\Ensurance\ensure;
-
 use Exception;
 
+use function Dgame\Ensurance\ensure;
+
 /**
  * Class BeamtenPensionRechner
  * @package Finanzrechner\Pension
  */
+
+class Pensionsberechnung
+{
+    public float $pensionsbetrag;
+    public bool $isMindestruhegehalt;
+
+    public function __construct(float $pensionsbetrag, bool $isMindestruhegehalt)
+    {
+        $this->pensionsbetrag      = $pensionsbetrag;
+        $this->isMindestruhegehalt = $isMindestruhegehalt;
+    }
+}
+
```

Comment:
In PHP und vielen anderen Sprachen gilt die Regel eine Klasse pro Datei.

Des Weiteren ist diese Klasse eigentlich nur ein simpler Container. Bestenfalls enthält dieser unveränderbare Properties, da er das Ergebnis einer konkreten Berechnung abbildet, die vorher stattgefunden haben sollte. So wie jetzt hier, können `$pensionsbetrag` und `$isMindestruhegehalt` von außen verändert werden. Das kann man umgehen in dem man die beiden Properties private macht und stattdessen getter-Funktionen verwendet